### DISCUSSION TITLE
Pipeline technical meetings of September 20

### DISCUSSION THREAD

**Discussion by rdeltour** [daisy/pipeline#791] [Meetings]
Dear all,

The next set of Pipeline technical meetings are scheduled on September 20, 2024.

You can attend any of these meetings depending on your time zone. The
timings are:

September 20, at 9 UTC: This time is friendly for Europe, Asia, and Australia.
September 20, at 15 UTC: This time is friendly for America and Europe.

[The Zoom details are the same for both meetings.](https://us02web.zoom.us/j/84980183752?pwd=TkRvUTJLbG8zbVhXdE9HQXlVc3RCUT09)

We will notably discuss the work plan for the coming year, to be presented at the next DAISY board meeting.

**Comment by bertfrees** [comment]
Hi all,

Below is a short list of things we have selected for the 2025 work plan.

This list is partly based on feedback we got from you (through the 2023 Survey and other channels), and partly on what we believe is important for the quality and long-term goals of the project. The list also includes some work items from this year's work plan that we were not able to finish. I have left out some "evergreen" items that we include every year. (Things like reacting to critical bugs and high-priority feature requests, and other maintenance work.)

The list is not final yet. We would like to discuss it with you, and adjust where needed.

### New scripts

- DTBook to eBraille script
- Word to EPUB 3 script

### Other added/improved functionality

- Focus on improving text-to-speech support (details to be determined)
- Focus on improving the usability of the desktop application (details to be determined)
- Distribute the command line tool with the desktop application
- Make braille CSS compatible with standard CSS (related to eBraille support, and building a repository of braille CSS style sheets)
- Support for Amazon Polly (contribution by ONCE)
- Align ODT and Word scripts with each other

### Save-as-DAISY add-in for Word

- Continue to improve the Microsoft Word add-in (details to be determined)
- Continue to align the add-in with the standalone Pipeline 2

### Pipeline 1

- Experiment with a Pipeline 1 backend for the Pipeline 2 API, allowing Pipeline 1 scripts to be exposed in the Pipeline 2 GUI, and the Pipeline 1 GUI to be discontinued.[^1]

### Webservice improvements

- Make use of the WebSocket protocol in the web service API to simplify client software.[^1]
- Support JSON as format for requests and responses in the web service API.[^1]
- Explore the possibility of supporting query parameters as an alternative to the job request document.

### General issues (related to maintenance, external contributions)

- Start extracting parts of the code in standalone libraries.[^1]
- Continue to make the build system more contributor- (and maintainer-) friendly.

[^1]: These were part of this year's work plan, but are not finished.



Hope to talk to you on Friday!

Bert
